index.js

```diff
@@ -293,9 +284,12 @@ This will become a hard error in the future.`,
         ms.prepend(opts.loader);
       }
 
+      const newCode = ms.toString();
+      const hasCodeChanged = code !== newCode;
```

Comment:
By this time we're guaranteed to modify the code, at least via unconditional `ms.remove` and `ms.prepend` just before this check, no?

Reply:
@RReverser not necessarily - the unconditional remove/prepend only modify the code if you're using a custom amd function name. Majority of our project's files fell into the `hasCodeChanged === false` path which is how this made such a difference to our bundle time when using this plugin.

Reply:
Hm, ok. Either way, as mentioned above - can you please split those changes off into a separate PR? Them being in one PR & commit makes it harder to reason about them.

Reply:
Unfortunately I'm no longer using the plugin - would you like to take over these changes? FWIW, the source map change only touches 5 lines at the end of renderChunk, if that helps clarify

Reply:
Ok, I'll look into that. Thanks for the PR.